src/backend/src/services/organizations.service.ts

```diff
@@ -56,6 +57,29 @@ export default class OrganizationsService {
     return newLinks;
   }
 
+  /**
+   * sets an organizations images
+   * @param submitter the user who is setting the images
+   * @param organizationId the organization which the images will be set up
+   * @param images the images which are being set
+   */
+  static async setImages(images: Express.Multer.File[], submitter: User, organizationId: string) {
+    if (!(await userHasPermission(submitter.userId, organizationId, isAdmin)))
+      throw new AccessDeniedAdminOnlyException('update images');
+
+    const imageData = await Promise.all(images.map((file: Express.Multer.File) => uploadFile(file)));
+
+    const newImages = await prisma.organization.update({
+      where: { organizationId },
+      data: {
+        interestedinApplyingImage: imageData[0].id,
```

Comment:
Continued from above: it would be nice to not have specific indices corresponding to fields (think "magic numbers")
Additionally it will make things easier down the road if we want more images (and relying on images being uploaded and retrieved in a consistent order is not great)